xml/vk.xml

```diff
@@ -6207,6 +6226,9 @@ server.
             <param externsync="true"><type>VkDescriptorSet</type> <name>descriptorSet</name></param>
             <param><type>VkDescriptorUpdateTemplate</type> <name>descriptorUpdateTemplate</name></param>
             <param noautovalidity="true">const <type>void</type>* <name>pData</name></param>
+            <implicitexternsyncparams>
+                <param>the sname:VkDescriptorPool that pname:descriptorSet was allocated from</param>
+            </implicitexternsyncparams>
```

Comment:
Similar comment to vkUpdateDescriptorsets.

Other than these two, the rest look fine to me.